tokens/bankSupplyTokens/mainnet.json

```diff
@@ -4120,7 +4120,7 @@
     "tokenVerification": "internal"
   },
   {
-    "hash": "/021FDD63F6D8DA6998A93DD25A72BD18421604A50819D01932136E934F9A26C4",
+    "hash": "021FDD63F6D8DA6998A93DD25A72BD18421604A50819D01932136E934F9A26C4",
```

Comment:
manually removed the forward slash in front of these "hash" properties. had to do manually because the list does not get written over if the ibc tokens already have cache denom trace. lmk if any issue with this